data/announcements/scheduled.yaml

```diff
@@ -71,4 +71,26 @@ announcements:
     <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon EU 2021</a> <em>virtual</em>.
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
-    May 4 - May 7, 2021.
\ No newline at end of file
+    May 4 - May 7, 2021.
+
+- name: Kubecon 2021 NA
+  startTime: 2021-11-10T00:00:00
+  endTime: 2021-10-16T00:00:00
+  style: "background: linear-gradient(90deg, rgb(7, 132, 111) 0%, rgb(54, 214, 183) 100%)" 
+  title: |
+    <img src="https://events.linuxfoundation.org/wp-content/uploads/2020/11/KubeCon_NA_2021_web_web-logo-white-1.svg" style="float: right; height: 80px;" />
+    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/"> KubeCon + CloudNativeCon North America 2021</a> <em>Los Angeles, California + Virtual</em>.
```

Comment:
We should copy the SVG image into _this_ Git repository, to avoid an extra cross-site image load on every page serve.

Reply:
If you're not sure how to do that, again, it's OK to ask in the [#sig-docs](https://kubernetes.slack.com/messages/sig-docs) channel on Kubernetes Slack.
Just an extra note on that: the next Kubernetes release comes out _this week_ so people might be a bit busy right now. We'll still try to find time to help!